src/builders/util.js

```diff
@@ -0,0 +1,5 @@
+export function findNearestElement(view, selector) {
+  if (!view || view.matches("html")) return null;
+  if (!selector) return null;
+  return document.querySelector(selector) || findNearestElement(view.parentElement, selector);
```

Comment:
No, because, when finding the content targeted by a validation constraint state property or the content targeted by the options property, the odds that the content is inside the input are almost zero.